jck/jck.mk

```diff
@@ -61,6 +61,10 @@ OTHER_OPTS=
 # if JDK_IMPL is openj9 or ibm
 ifneq ($(filter openj9 ibm, $(JDK_IMPL)),)
  OTHER_OPTS=-Xtrace:maximal=all{level2}
+# if JDK_VERSION is 8
+ifneq ($(filter 8, $(JDK_VERSION)),) 
```

Comment:
Also, your logic didn't add `-Xfuture`. As mentioned in the description of the [issue](https://github.com/AdoptOpenJDK/openjdk-tests/issues/1754) for this pr, -Xfuture should be added for all JDK versions, and `--enable-preview` should be added when JDK_VERSION != 8. 

To achieve that, you may want to make the following updates to your PR: 

1) Instead of an empty string, set default value of OTHER_OPTS=-Xfuture at line [60](https://github.com/AdoptOpenJDK/openjdk-tests/blob/08a8e1bc995c7e33cb0632b1bb2e6f5d3664a00e/jck/jck.mk#L60). 
2) Replace the subsequent value assignments (`=`) to `OTHER_OPTS` with increment (`+=`) so that we do overwrite values set previously. For example, `OTHER_OPTS=-Xtrace:maximal=all{level2}` - this line should become `OTHER_OPTS+=-Xtrace:maximal=all{level2}`.

Reply:
Thank you for the review! I made the changes. Have a good weekend!